### PR TITLE
CI: merge with MINGW-packages

### PR DESCRIPTION
**Pull request by jeremyd2019**

### CODE REVIEW

**Comment by jeremyd2019** [comment]
Test run: https://github.com/jeremyd2019/MSYS2-packages/actions/runs/1204607008

**Comment by jeremyd2019** [comment]
I didn't try to look at the python 'check' job, I don't really know if that would translate to MSYS2.

File: .ci/ci-library.sh [hunk 1]
```diff
@@ -114,29 +73,18 @@ execute(){
...
+# Get changed packages in correct build order
 list_packages() {
-    local _packages
-    _list_changes _packages '*/PKGBUILD' '%/PKGBUILD' --pretty=format: --name-only || return 1
-    for _package in "${_packages[@]}"; do
-        local find_case_sensitive="$(find -name "${_package}" -type d -print -quit)"
-        test -n "${find_case_sensitive}" && packages+=("${_package}")
-    done
-    return 0
+    _as_list "packages" "*" "" "$($DIR/ci-get-build-order.py)"
```
Comment (jtanx) [inline .ci/ci-library.sh:83]: One thing I didn't realise when switching to use `_as_list`, it assumes that no output is a failure, but in this case it should actually be allowed (see line 27 in ci-build.sh) - that's why the build is failing right now.

Comment (jeremyd2019) [reply .ci/ci-library.sh:83]: msys2/MINGW-packages#9545

**Pull request merged**